standard-questions.md

```diff
@@ -132,7 +132,7 @@ __Where is your solution's source hosted?__
             </td>
         </tr>
         <tr>
-            <td valign="top">8. Open Standards & Best Practices</td>
+            <td valign="top">8. Standards & Best Practices</td>
```

Comment:
Indicator should be "Adherence to Standards & Best Practices" from the website.

Reply:
@BolajiAyodeji We're using short names for the questionnaire as they will be displayed in the nomination form.

Reply:
@BolajiAyodeji We're using short names for the questionnaire as they will be displayed in the nomination form.